### PR TITLE
hevm: default smttimeout to 20s

### PR DESCRIPTION
**Pull request by xwvvvvwx**
unlimited by default is probabily a little user unfriendly

### CODE REVIEW

File: src/hevm/hevm-cli/hevm-cli.hs [hunk 1]
```diff
@@ -373,7 +373,7 @@ equivalence cmd =
        Just sig' -> do method' <- functionAbi sig'
                        return $ Just (view methodSignature method', snd <$> view methodInputs method')
 
-     void . runSMTWithTimeOut (solver cmd) (smttimeout cmd) . query $
+     void . runSMTWithTimeOut (solver cmd) (fromMaybe 5000 (smttimeout cmd)) . query $
```
Comment (MrChico) [inline src/hevm/hevm-cli/hevm-cli.hs:376]: if you set the default in `runSMTWithTimeOut` we don't have to set it in two places

Comment (xwvvvvwx) [reply src/hevm/hevm-cli/hevm-cli.hs]: done.

**Comment by xwvvvvwx** [comment]
wondering if we want to have some syntax for setting an unlimited timeout? maybe `--smttimeout=0`?

**Comment by MrChico** [comment]
yeah, I was thinking about that too, but maybe one can just set a ridiculously high timeout in that case instead

**Comment by MrChico** [comment]
I suppose you have to increase the timeout for the `factor` test for it to work

**Comment by xwvvvvwx** [comment]
hmmm, maybe 5000 as a default is too short in that case?

**Comment by MrChico** [comment]
Maybe 20s?

**Pull request merged**